core/src/main/java/com/radixdlt/ledger/LedgerUpdate.java

```diff
@@ -64,58 +64,23 @@
 
 package com.radixdlt.ledger;
 
-import com.google.common.collect.ClassToInstanceMap;
 import com.radixdlt.consensus.LedgerProof;
-import com.radixdlt.consensus.bft.BFTValidatorSet;
+import com.radixdlt.consensus.epoch.EpochChange;
+import com.radixdlt.statecomputer.commit.CommitSummary;
 import com.radixdlt.transactions.RawLedgerTransaction;
 import java.util.List;
-import java.util.Objects;
 import java.util.Optional;
 
-public final class LedgerUpdate {
-  private final LedgerExtension ledgerExtension;
-  // FIXME: Easiest way to implement this part for now
-  private final ClassToInstanceMap<Object> output;
+public record LedgerUpdate(
+    CommitSummary commitSummary,
+    LedgerExtension ledgerExtension,
+    Optional<EpochChange> maybeEpochChange) {
```

Comment:
```suggestion
    Optional<EpochChange> epochChange) {
```
("maybe" feels like an empty word, and it is typically skipped in method names - it's hard to miss the type in java)